assets/js/components/Dashboard/PaMessaging/NewPaMessagePage/NewPaMessagePage.tsx

```diff
@@ -224,13 +257,15 @@ interface ReviewAudioButtonProps {
   audioPlaying: boolean;
   audioReviewed: boolean;
   disabled?: boolean;
+  showErrorIcon?: boolean;
   onClick: () => void;
 }
 
 const ReviewAudioButton = ({
   audioPlaying,
   audioReviewed,
   disabled,
+  showErrorIcon,
```

Comment:
Since I think we're only ever in one of the "playing", "reviewed", or "errored" states at a time, maybe those three props could be replaced with a single one, defined as an enum? Or does it make sense to possibly have multiple of these states at once?

Reply:
Took a stab at this. Hopefully I didn't make it too complicated.